### ISSUE TITLE
[SF_ACC] Nested Tab Container NVDA Missing Instructions

### ISSUE THREAD

**Issue by IrvinDelgado** [SAP/ui5-webcomponents#6419]
### **Bug Description**
In NVDA a blind user would not know that the container they are on has nested containers

### **Expected Behavior**
Some sort of Message allowing blind users to know that there are subcontainers

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/TabContainer/ 
2. Open NVDA 
3. Focus on the Nested TabContainers Section
4. Press the right error to focus on the Nested TabContainer
5. Notice that there are no instructions to let the user know that there are subcontainers

### **Isolated Example**
If possible, please provide an isolated example by forking [CodeSandbox](https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html).

### **Context**
- UI5 Web Components version: 1.10.0
- OS/Platform: Windows
- Browser: Chrome
- Affected component: Tab Container

### **Log Output / Stack Trace / Screenshots**
![Screenshot 2023-02-03 at 3 03 29 PM](https://user-images.githubusercontent.com/62973847/216727578-09674f98-25e1-4b7e-8d82-94a00801891b.png)

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

**Comment by dobrinyonkov** [comment]
Hello @SAP/ui5-webcomponents-topic-rd,

using a screen reader (JAWS or NVDA), there is no announcement to inform that the tab has subitems. 

Could you please take a look into that issue?

Kind Regards,
Dobrin

**Comment by TeodorTaushanov** [comment]
Hi @IrvinDelgado,

The accessibility of the ui5-tabcontainer, when they are subitems, is improved in the main branch.

Best,
Teo
